skimage/_shared/testing.py

```diff
@@ -201,15 +201,28 @@ def setup_test():
 
     warnings.simplefilter('error')
 
+    # do not error on specific warnings from the skimage.io module
+    warnings.filterwarnings(
+        'default', message='TiffFile:', category=DeprecationWarning,
```

Comment:
what does this do? we need this?

Reply:
I see so this should really be open as an issue, for us to address the warnings appropriatly.

Reply:
Yes, I opened a new issue at #5337 just now where the specific warnings that I observed locally are listed. 3 are deprecated parameters names for `tifffile` and the rest are `ResourceWarning`s about unclosed image files.